src/main/java/it/unisa/Amigo/documento/service/DocumentoService.java

```diff
@@ -9,7 +9,7 @@
 public interface DocumentoService {
     Documento addDocumento(MultipartFile file);
     Documento updateDocumento(Documento documento);
-    Documento findDocumento(int idDocumento);
-    List<Documento> searchDocumenti(String nameDocumento);
+    Documento findDocumento(Integer idDocumento);
```

Comment:
```suggestion
    Documento findDocumentoById(Integer idDocumento);
```
Quando applicate il cambio nome usare la funzionalità di refactoring su intellij, non cambiate il nome a mano.